config/initializers/recaptcha.rb

```diff
@@ -0,0 +1,4 @@
+Recaptcha.configure do |config|
+  config.site_key = ENV['RECAPTCHA_SITE_KEY']
```

Comment:
It is useful to add these ENV variables to the file `.env.example` so we know we need them.